plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixMetadata.java

```diff
@@ -83,19 +84,32 @@ public JdbcTableHandle getTableHandle(ConnectorSession session, SchemaTableName
     }
 
     @Override
-    public ConnectorTableMetadata getTableMetadata(ConnectorSession session, ConnectorTableHandle table)
+    public ConnectorTableBasicMetadata getTableBasicMetadata(ConnectorSession session, ConnectorTableHandle table)
     {
-        return getTableMetadata(session, table, false);
+        JdbcTableHandle handle = (JdbcTableHandle) table;
+        return new ConnectorTableBasicMetadata(
+                getSchemaTableName(handle),
+                getColumnMetadata(session, false, handle).stream()
+                        .map(ColumnMetadata::toBasicMetadata)
+                        .collect(toImmutableList()));
     }
 
-    private ConnectorTableMetadata getTableMetadata(ConnectorSession session, ConnectorTableHandle table, boolean rowkeyRequired)
+    @Override
+    public ConnectorTableMetadata getTableMetadata(ConnectorSession session, ConnectorTableHandle table)
     {
         JdbcTableHandle handle = (JdbcTableHandle) table;
-        List<ColumnMetadata> columnMetadata = phoenixClient.getColumns(session, handle).stream()
+        return new ConnectorTableMetadata(
+                getSchemaTableName(handle),
+                getColumnMetadata(session, false, handle),
+                phoenixClient.getTableProperties(session, handle));
+    }
+
+    private List<ColumnMetadata> getColumnMetadata(ConnectorSession session, boolean rowkeyRequired, JdbcTableHandle handle)
```

Comment:
order of parameters differs from https://github.com/trinodb/trino/pull/6975/commits/eaf66717f28023799cdc2552b882587ba0cb6eb5#diff-b2770586e716078fd17f177ce4cd0868628f704bc0cd81f3fb5f3d4def54e882L91

`rowkeyRequired` is always `false`.